rocky/rocky/views/finding_list.py

```diff
@@ -102,20 +121,33 @@ def order_by(self) -> Literal["score", "finding_type"]:
     def sorting_order(self) -> Literal["asc", "desc"]:
         return "asc" if self.request.GET.get("sorting_order", "") == "asc" else "desc"
 
+    def get_object_list_settings_form_kwargs(self):
+        data = self.request.POST if self.request.POST else None
+
+        return {"organization": self.organization, "data": data}
```

Comment:
The name doesn't make sense. Also, if there's only a single reference to this, I would inline it